frontend/templates/cucumber/.github/workflows/browser.yml

```
@@ -0,0 +1,34 @@
+name: Run Browser Tests
+
+on:
+  pull_request:
+  workflow_dispatch:
+
```

```suggestion
permissions: {}

```
Override the default permissions which are too permissive